ludwig/schema/features/text_feature.py

```diff
@@ -235,5 +234,5 @@ class LLMTextDefaultsConfig(TextInputFeatureConfigMixin, TextOutputFeatureConfig
 
     loss: BaseLossConfig = LossDataclassField(
```

Comment:
Correct, that's right. Any other loss would be wrong for LLM fine-tuning. At the moment, the `LossDataclassField` allows all loss metrics in the registry. I've added a todo comment for now so that we remember this is something we want to take up in the future.

Reply:
@justinxzhao not 100% sure that will work since it won't return the schema for the loss function itself, which is what the LossDataclassField will do at run-time when we make the call to `get_schema_from_registry`.

Reply:
right, if loss is meant to be a nested config rather than a string it would not be advisable to use `StringOptions` (unless you want to push the loss config initialization logic to the `post_init` hook for `LLMTextDefaultsConfig` or some other supra-layer).